test/transpile.jl

```diff
@@ -1756,9 +1756,7 @@ end
 
     connectivities_and_targets = [
         (LineConnectivity(12), (1, 2))
-        (LineConnectivity(12, collect(9:12)), (1, 2))
```

Comment:
I suggest leaving these in, if only to ensure the excluded positions have no effect